### DISCUSSION TITLE
Paste clipboard screenshot feature request

### DISCUSSION THREAD

**Discussion by drwbns** [cline/cline#592] [Feature Requests]
Hey guys. I'm on a Windows machine so I'm not sure if MacOS is already supporting this but it would be really handy for fast code iteration to be able to screenshot and paste directly into Cline's chat withoutout having to save the jpg to disk first. I use a screenshot program (Lightshot) that allows me to just clip a window area that I would love to just paste into chat if possible. Any way to do this?

**Comment by ctwhome** [comment]
It works on macos indeed.

**Comment by pleabargain** [comment]
pasting screenshots works for me on win11 as of 2025 APRIL
